File: in_Python/1295 Find Numbers with Even Number of Digits.py
class Solution(object):
    def helper(self, t):
        temp = 0
        while t != 0:
            t /= 10
            temp += 1
        return temp % 2 == 0
    
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        res = 0
        for x in nums:
            if self.helper(x):
                res += 1
        return res

        

File: in_Java/1295 Find Numbers with Even Number of Digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int res = 0;
        for(int i : nums){
            if(i > 9 && helper(i)) res++;
        }
        return res;
    }
    private boolean helper(int n){
        int t = 0;
        while(n != 0){
            n /= 10;
            t++;
        }
        return t % 2 == 0;
    }
}